src/util/BaseLoginRouter.js

```diff
@@ -113,6 +114,11 @@ function (Okta, Backbone, xdomain, RefreshAuthStateController, Settings, Header,
       this.listenTo(this.appState, 'navigate', function (url) {
         this.navigate(url, { trigger: true });
       });
+
+      // triggering event up the chain
+      this.listenTo(Okta.Controller.prototype, 'pageRenderedEvent', function (className) {
```

Comment:
Looking at this now, I think we should go the "use bubble property route", i.e. it's not as nice to listen to the prototype here. Feels better to listen to an instance when it's created.